.eslintcache

```diff
@@ -0,0 +1 @@
+[{"/Users/comalvirdi/h4i2/rise-stand-strong/src/index.js":"1","/Users/comalvirdi/h4i2/rise-stand-strong/src/App.js":"2","/Users/comalvirdi/h4i2/rise-stand-strong/src/components/Main/Main.jsx":"3","/Users/comalvirdi/h4i2/rise-stand-strong/src/components/LoginPage/LoginPage.jsx":"4"},{"size":341,"mtime":1610414673528,"results":"5","hashOfConfig":"6"},{"size":192,"mtime":1610414673527,"results":"7","hashOfConfig":"6"},{"size":272,"mtime":1610414673528,"results":"8","hashOfConfig":"6"},{"size":2596,"mtime":1611357767890,"results":"9","hashOfConfig":"6"},{"filePath":"10","messages":"11","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"11w8nzf",{"filePath":"12","messages":"13","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"14","messages":"15","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"16","messages":"17","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"/Users/comalvirdi/h4i2/rise-stand-strong/src/index.js",[],"/Users/comalvirdi/h4i2/rise-stand-strong/src/App.js",[],"/Users/comalvirdi/h4i2/rise-stand-strong/src/components/Main/Main.jsx",[],"/Users/comalvirdi/h4i2/rise-stand-strong/src/components/LoginPage/LoginPage.jsx",[]]
```

Comment:
Can you add `.eslintcache` to the gitignore? Don't think we need to push this to master

Reply:
Yeah